double/e_rem_pio2.c

```diff
@@ -47,7 +47,10 @@ pio2_3t =  8.47842766036889956997e-32; /* 0x397B839A, 0x252049C1 */
 #ifdef _MSC_VER
 static __forceinline
 #else
-static __inline __always_inline
```

Comment:
A better change would be to replace this line with `static __inline __attribute__((always_inline))` instead which should be supported for both clang and gcc.

Reply:
Well spotted, works on Mac too so changing it